config-connector/solutions/projects/kpt/owned-project/README.md

```diff
@@ -0,0 +1,66 @@
+Owned Project
+==================================================
+
+# NAME
+
+  owned-project
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create
+  a project in a folder, binding an IAM member
+  as project owner.
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
```

Comment:
Let's make sure the text before each command be a proper explanation/description of the command. Here, I think it would be better if we say something like "Fetch the kpt package of the solution".